src/utils/pisaDataBinaryToUrl.ts

```diff
@@ -0,0 +1,31 @@
+import { pisaClient, pisaUrl } from '../Data/pisaClient'
+import { FullDataBinary } from './dataBinaryToUrl'
+
+export async function pisaDataBinaryToUrl(binary: FullDataBinary) {
```

Comment:
* [ ] Missing return type. However, as long as dataBinaryToUrl is the only importing module, we could move it there as a private helper.

Reply:
Same as https://github.com/Scrivito/scrivito-portal-app/pull/381#discussion_r1578102346,